### DISCUSSION TITLE
bug report template?

### DISCUSSION THREAD

**Discussion by cralwalker** [WillCodeForCats/solaredge-modbus-multi#49] [General]
Pardon a newbee, I experience the same issue as @Robby-Swartenbroekx in issue #48 "Inverter device ID 1 not found. #48"
I am happy to provide all information needed in a bug report, if I knew where to find the template.

**Comment by WillCodeForCats** [comment]
When you open an issue, click the green "get started" button in the bug report section.

<img width="1097" alt="Screen Shot 2022-07-16 at 8 49 30 AM" src="https://user-images.githubusercontent.com/48533968/179362144-a4cec2af-f4f5-41d1-8978-9aa7edb811c4.png">
